shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -47,7 +50,7 @@
 
 @Config(manifest = Config.NONE)
 @RunWith(RobolectricTestRunner.class)
-@TargetApi(28)
+@TargetApi(30)
```

Comment:
If you do this, I think we'd want another API 28 test to make sure the existing logic still works. Since it's gonna take a really long time for a significant number of our users to be on R. We need to make sure we don't regress on <R.